src/overlays/actors/ovl_En_Dekubaba/z_en_dekubaba.h

```diff
@@ -6,9 +6,29 @@
 
 struct EnDekubaba;
 
+typedef void (*EnDekubabaActionFunc)(struct EnDekubaba*, GlobalContext*);
+
+typedef enum {
+    /* 0 */ DEKUBABA_NORMAL,
+    /* 1 */ DEKUBABA_BIG
+} DekuBabaType;
+
 typedef struct EnDekubaba {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x2CC];
+    /* 0x014C */ Vec3f bodyPartsPos[4];
+    /* 0x017C */ SkelAnime skelAnime;
+    /* 0x01C0 */ EnDekubabaActionFunc actionFunc;
+    /* 0x01C4 */ char pad[0x2];
+    /* 0x01C6 */ s16 timer;
+    /* 0x01C8 */ s16 targetSwayAngle;
+    /* 0x01CA */ s16
+        stemSectionAngle[3]; // Used to calculate the position of the stem sections and head with spherical trigonometry
```

Comment:
Can you put this in the same line?

Reply:
'Twas the formatter what done it...

Reply:
But `format.h` doesn't try to format headers :thinking:

Reply:
yea you can run the formatter on headers manually if you want, but make sure to fix stuff like this. the formatter will also tend to put the "size" comment on the actor struct aligned way to the right. when we manually keep it on the left right after the semicolon